src/scripts/elasticsearch-ubuntu-install.sh

```diff
@@ -332,6 +337,17 @@ install_azure_cloud_plugin()
     fi
 }
 
+install_additional_plugins()
+{
+    log "[install_additional_plugins] Installing additional plugins"
+    for PLUGIN in $(echo $INSTALL_ADDITIONAL_PLUGINS | tr ";" "\n")
```

Comment:
Reinstalling an installed plugin produces a message that the plugin is already installed.  It would be difficult to identify the plugins already installed because the naming conventions are different between `plugin --list` and `plugin --install`.